### PR TITLE
Rebase PR 1524

### PR DESCRIPTION
**Pull request by vsn4ik**
Rebase #1524.
Unfortunately for UTC+0 this test is always performed.

### CODE REVIEW

**Comment by acrobat** [comment]
Hmm so we need to find a way to have the tests run with different timezone offsets..

**Comment by Azaret** [comment]
If we don't want to implement timepicker until way later, we should get rid of timezone altogether.
Implying get rid of Date object.

I'm not against one day having time support. But in the short term, I think it could ease our development, so we can move on in many issues, focus on other features, and come back on time support later.

**Comment by acrobat** [comment]
Yes time support is something for atleast version 2.0 but could be moved back even further. Maybe we should indeed focus now on making the datepicker not depend on timezone etc.

**Comment by vsn4ik** [comment]
I try to play with `process.env.TZ`

**Comment by Azaret** [comment]
@acrobat Well, for sure we need to not have `new Date()` everywhere at least. Then doing a custom date object is not that hard.
Well to be honest I'm not sure if doing our own object will be a good idea for sure, all I'm sure of is that we are having a lot of troubles by using Date. It feels like a black box that can do unexpected things and we can't control very well.

**Comment by vsn4ik** [comment]
PR updated.

**Comment by acrobat** [comment]
Oh great so now we can run 2 test suits? The normal `test` and `qunit:timezone`?

**Comment by vsn4ik** [comment]
Not, only `test`.

**Comment by acrobat** [comment]
Oh great, missed that! I will do final review soon!

File: js/bootstrap-datepicker.js [hunk 1]
```diff
@@ -919,8 +919,8 @@
 				tooltip;
 			if (isNaN(year) || isNaN(month))
 				return;
-			this.picker.find('.datepicker-days thead .datepicker-switch')
```
Comment (acrobat) [inline js/bootstrap-datepicker.js]: Why is the thead class removed here?

Comment (acrobat) [inline js/bootstrap-datepicker.js]: ping @vsn4ik

Comment (vsn4ik) [inline js/bootstrap-datepicker.js]: My version is more concise. Which option is faster, I do not know. I go back as it was.

Comment (acrobat) [inline js/bootstrap-datepicker.js]: Oh ok, if you want you can create a separate pr for that change! Just so we don't mix different change sets

**Comment by acrobat** [comment]
Thanks @vsn4ik!

**Pull request merged**